pyproject.toml

```diff
@@ -47,7 +47,7 @@ exclude = '''
         |palettes
         |_utils
```

Comment:
Seems like you formatted `plotting/_utils.py`, so you need to remove this line.

Reply:
Done